src/lib/tls/tls_policy.cpp

```diff
@@ -568,9 +568,11 @@ void print_vec(std::ostream& o, const char* key, const std::vector<std::string>&
 void print_vec(std::ostream& o, const char* key, const std::vector<Group_Params>& v) {
    o << key << " = ";
    for(size_t i = 0; i != v.size(); ++i) {
-      o << group_param_to_string(v[i]);
-      if(i != v.size() - 1) {
-         o << ' ';
+      if(auto name = v[i].to_string()) {
+         o << name.value();
+         if(i != v.size() - 1) {
+            o << ' ';
+         }
```

Comment:
I guess, for the time being, we'll just have to either assert that `to_string()` never fails or conditionally unwrap and push_back into a temporary `std::vector<std::string>`. 😞

Reply:
Eh, std::vector it is.

I think we actually misbehaved in a similar way before; instead of returning an `optional`, to_string indicated failure by returning an empty string, so we'd get a double space. Which I think the parser should handle but IDK

Reply:
Changed to vector